tasks-by-users/samar4tech/01-e-commerce-website/style.css

```diff
@@ -0,0 +1,508 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+.container {
+  max-width: 1300px;
+  margin: auto;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+.navbar {
+  padding: 20px;
+  display: flex;
+  align-items: center;
+}
+.logo img {
+  width: 125px;
+}
+
+.navbar img[src^="images/cart"] {
+  width: 30px;
+  height: 30px;
+  margin-top: 12px;
+}
+nav {
+  flex: 1;
+}
+
+nav ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-end;
+}
+nav ul li {
+  padding-right: 20px;
+}
+
+a {
+  text-decoration: none;
+  color: #555;
+}
+p {
+  color: #555;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+.col-2 {
+  flex-basis: 50%;
+  min-width: 300px;
+}
+.col-2 img {
+  max-width: 100%;
+  padding: 50px 0;
+}
+.col-2 h1 {
+  font-size: 50px;
+  line-height: 60px;
+  margin: 25px 0;
+}
+.btn {
+  display: inline-block;
+  background-color: #ff523b;
+  color: #fff;
+  padding: 8px 30px;
+  margin: 30px 0;
+  border-radius: 30px;
+  transition: background 0.5s;
+}
+.header {
+  background: radial-gradient(#fff, #ffd6d6);
+}
+.header .row {
+  margin-top: 70px;
+}
+.btn:hover {
+  background-color: #563434;
```

Comment:
```suggestion
  background: #563434;
```

Reply:
specifically used only color property in the bg so opted background-color.Will take care from the next time.